index.html

```diff
@@ -13,13 +13,2689 @@
       href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@300;400;700&display=swap"
       rel="stylesheet"
     />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
     <link rel="stylesheet" href="assets/css/custom.css" />
+
     <title>Newsweek</title>
   </head>
 
-  <body>
-    <header></header>
-    <main></main>
-    <footer></footer>
+  <body class="container-fluid px-0">
```

Comment:
Good job making the clone full width in smaller screens but you still need to put spacing on either side of your clone in larger screens(desktop).

| Source Page	| Clone Page |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/57726348/87222030-10a14c00-c38e-11ea-8940-03e9da33cd58.png)|![image](https://user-images.githubusercontent.com/57726348/87222035-1b5be100-c38e-11ea-8ef0-51876a7094c4.png)|

Reply:
Done.